Dynatrace/dynatrace-operator
# Installs Helm unittest plugin explicitly

# Description

The original script installing the Helm unittest plugin is using the GitHub API which is rate limiting how often the plugin can be downloaded. 
These changes implement a more explicit installation of the plugin, so the CI does not break whenever the API would be rate limiting installations.

## How can this be tested?

* Run `make prerequisites/helm` to check if the installation script works
* Run `make test/helm` to check if the plugin was installed correctly
* If possible, run `make prerequisites/helm` on an ARM machine to check if the architecture selection works correctly


## Checklist
~~- [ ] Unit tests have been updated/added~~
